pyproject.toml

```diff
@@ -88,12 +88,11 @@ exclude = [
 [tool.ruff.format]
 quote-style = 'single'
 
-[tool.ruff.per-file-ignores]
+[tool.ruff.lint.per-file-ignores]
 # allow asserts in test files
 '*test_*.py' = ['S101']
 
 [tool.ruff.lint]
-ignore-init-module-imports = true
```

Comment:
this seems to be deprecated in new ruff versions, so we should also modify the template, where this statement originated from.
Can you make an issue for `EasyProjectTemplate` project, so we can modify it at some point?

Reply:
https://github.com/EasyScience/EasyProjectTemplate/issues/6